code/controllers/subsystems/configuration.dm

```diff
@@ -0,0 +1,151 @@
+SUBSYSTEM_DEF(configuration)
```

Comment:
Not sure if this should be a subsystem? Because config is actually loaded before master inits inside world procs. Things like fps, ticklag, sql sever login and more.

Reply:
Sounds good! 
The vv stuff I thought was to prevent changing settings that would have no effects at runtime?